agent/plugin/error.go

```diff
@@ -41,13 +40,18 @@ func (e *DeprecatedNameErrors) Errors() []DeprecatedNameError {
 		return errs[i].old < errs[j].old
 	})
 
-	return errs
+	out := make([]error, 0, len(errs))
+	for i := range errs {
+		out = append(out, &errs[i])
+	}
```

Comment:
I guess the alternative is to make `errs` have type `[]error`, then type-assert everything in the `sort.Slice` closure...

Hang on, does anything use the order of `e.errs`?

Reply:
I think it's nice for the user. I wrote tests for it, so they will break

Reply:
Hmm. Maybe I didn't 🤷‍♂️. Still think it's nice for the user.